src/main/content/_assets/js/builds.js

```diff
@@ -360,15 +361,16 @@ function render_builds(builds, parent) {
                                 build.version.lastIndexOf('.') + 1
                             ),
                             10
-                        );  
+                        );
                         if (package_name.indexOf('jakartaee10') > -1) {
-                            // 23.0.0.2 and higher should hav EE10 instead of EE9
+                            // 23.0.0.3 and higher should have EE10 instead of EE9
                             package_column =
                                     '<td headers=\'' +
                                     tableID +
-                                    '_package\'>Jakarta EE 10</td>';
+                                    '_package\'>Jakarta EE 10 <img class="info_tooltip" src="/img/information_downloads_table.svg" alt="For convenience, this package also includes features that enable MicroProfile 6"/> '+
+                                    '<p class="tooltip_text" style="display:none;">For convenience, this package also includes features that enable MicroProfile 6.</p></td>';
                         } else if (package_name.indexOf('jakartaee9') > -1) {
-                            // 21.0.0.12 to 23.0.0.2 should be labled "Jakarta EE 9"
+                            // 21.0.0.12 to 23.0.0.3 should be labled "Jakarta EE 9"
```

Comment:
23.0.0.3 says Jakarata EE 10, so i think the comment 21.0.0.12 to 23.0.0.2 would still be accurate